api/tests/unit_tests/core/test_model_manager.py

```diff
@@ -55,6 +57,10 @@ def incr(key):
         start_index += 1
         return start_index
 
+    fake_redis_client = redis.Redis(host="localhost", port=6379, db=0)
```

Comment:
You are pinning details the hardcoded values here. Could you change to use the constructor in order to align to the settings and test coverage?

Reply:
Okay, I have submitted the code according to your suggestions，thanks